settings/config.py

```diff
@@ -86,3 +86,6 @@ def get_reader_fetch_freq(self):
 
   def get_indexer_retrain_freq(self):
     return self.config["indexer"]["retrain_frequency"]
+
+  def get_consumer_group_mode(self):
+    return self.config["mode"]["consumer_group_mode"]
```

Comment:
where will this be in `app-config.json`?  this should be in connectors.readers["kafka_reader"] block.

Reply:
right now i have added an extra field named _"mode"_. and the format is shown below

>  "mode": {
>     "consumer_group_mode": "hostname"
>   }